draft-enghardt-panrg-path-properties.md

```diff
@@ -86,7 +88,7 @@ Path element:
 : Either a node or a link.
 
 Path:
-: A sequence of adjacent path elements over which a packet can be transmitted, starting and ending with a node. Paths are time-dependent, i.e., the sequence of path elements over which packets are sent from one node to another may change frequently. A path is defined between two nodes. For multicast or broadcast, a packet may be sent by one node and received by multiple nodes. In this case, the packet is sent over multiple paths at once, one path for each combination of sending and receiving node. Note that an entity may have only partial visibility of the path elements that comprise a path, and entities may treat path elements at different levels of abstraction.
+: A sequence of adjacent path elements over which a packet can be transmitted, starting and ending with a node. Paths are time-dependent, i.e., the sequence of path elements over which packets are sent from one node to another may change frequently. A path is defined between two nodes. For multicast or broadcast, a packet may be sent by one node and received by multiple nodes. In this case, the packet is sent over multiple paths at once, one path for each combination of sending and receiving node. Note that an entity may have only partial visibility of the path elements that comprise a path, and entities may treat path elements at different levels of abstraction. For example, a path may be given as a sequence of physical nodes and the links connecting these nodes, or it may be given as a sequence of logical nodes such as a sequence of ASes or an Explicit Route Object (ERO). Similarly, the representation of a path and its properties, as it is known to a specific entity, may be complex and include details on the used physical layer technology, or it may be abstract and only include a route to a specific destination.
```

Comment:
I would change the last sentence to "Similarly, the representation of a path and its properties, as it is known to a specific entity, may be more complex and include details about the physical layer technology, or it may be more abstract and only consist of a specific source and destination."

Since saying "only include a route" does not necessarily mean that the route is abstract and also we didn't define what a route is.

Reply:
Agree, "only include a route" is not specific enough here.
What I meant here is the information that a specific destination is reachable. I think your proposed sentence works, because if a path consists of a specific source and destination, this implies that it is reachable, otherwise it wouldn't be a path according to our definition.
However, if we want to be really really clear, how about "only consist of a specific source and a destination which is known to be reachable from that source"?

Reply:
Ok, I changed the sentence